src/Debug/ShutdownHandler.php

```diff
@@ -31,13 +27,7 @@ public static function register($debug = true)
         self::$errorLevels = error_reporting();
 
         if (PHP_SAPI === 'cli') {
-            $consoleHandler = function (\Exception $e) {
-                $app = new Application('Bolt CLI', Version::VERSION);
-                $output = new ConsoleOutput(OutputInterface::VERBOSITY_DEBUG);
-                $app->renderException($e, $output);
-                ob_clean();
-            };
```

Comment:
Add to `Application::initProviders()` at the end:
```
throw new \Exception('Koala');
```

`release/3.3`:

```
$ ./app/nut database:check -v

               
  [Exception]  
  Koala        
               

Exception trace:
 () at [snip]/src/Application.php:243
 Bolt\Application->initProviders() at [snip]/src/Application.php:119
 Bolt\Application->initialize() at [snip]/src/Application.php:64
 Bolt\Application->__construct() at [snip]/app/bootstrap.php:172
 Bolt\{closure}() at n/a:n/a
 call_user_func() at [snip]/app/bootstrap.php:213
 require() at [snip]/app/nut:14

```

This commit:

```
$ ./app/nut database:check -v

<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8" />
        <meta name="robots" content="noindex,nofollow" />
        <style>
            /* Copyright (c) 2010, Yahoo! Inc. All rights reserved. Code licensed under the BSD License: http://developer.yahoo.com/yui/license.html */
            html{color:#000;background:#FFF;}body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,code,form,fieldset,legend,input,textarea,p,blockquote,th,td{margin:0;padding:0;}table{border-collapse:collapse;border-spacing:0;}fieldset,img{border:0;}address,caption,cite,code,dfn,em,strong,th,var{font-style:normal;font-weight:normal;}li{list-style:none;}caption,th{text-align:left;}h1,h2,h3,h4,h5,h6{font-size:100%;font-weight:normal;}q:before,q:after{content:'';}abbr,acronym{border:0;font-variant:normal;}sup{vertical-align:text-top;}sub{vertical-align:text-bottom;}input,textarea,select{font-family:inherit;font-size:inherit;font-weight:inherit;}input,textarea,select{*font-size:100%;}legend{color:#000;}

            html { background: #eee; padding: 10px }
            img { border: 0; }
            #sf-resetcontent { width:970px; margin:0 auto; }
                        .sf-reset { font: 11px Verdana, Arial, sans-serif; color: #333 }
            .sf-reset .clear { clear:both; height:0; font-size:0; line-height:0; }
            .sf-reset .clear_fix:after { display:block; height:0; clear:both; visibility:hidden; }
            .sf-reset .clear_fix { display:inline-block; }
            .sf-reset * html .clear_fix { height:1%; }
            .sf-reset .clear_fix { display:block; }
            .sf-reset, .sf-reset .block { margin: auto }
            .sf-reset abbr { border-bottom: 1px dotted #000; cursor: help; }
            .sf-reset p { font-size:14px; line-height:20px; color:#868686; padding-bottom:20px }
            .sf-reset strong { font-weight:bold; }
            .sf-reset a { color:#6c6159; cursor: default; }
            .sf-reset a img { border:none; }
            .sf-reset a:hover { text-decoration:underline; }
            .sf-reset em { font-style:italic; }
            .sf-reset h1, .sf-reset h2 { font: 20px Georgia, "Times New Roman", Times, serif }
            .sf-reset .exception_counter { background-color: #fff; color: #333; padding: 6px; float: left; margin-right: 10px; float: left; display: block; }
            .sf-reset .exception_title { margin-left: 3em; margin-bottom: 0.7em; display: block; }
            .sf-reset .exception_message { margin-left: 3em; display: block; }
            .sf-reset .traces li { font-size:12px; padding: 2px 4px; list-style-type:decimal; margin-left:20px; }
            .sf-reset .block { background-color:#FFFFFF; padding:10px 28px; margin-bottom:20px;
                -webkit-border-bottom-right-radius: 16px;
                -webkit-border-bottom-left-radius: 16px;
                -moz-border-radius-bottomright: 16px;
                -moz-border-radius-bottomleft: 16px;
                border-bottom-right-radius: 16px;
                border-bottom-left-radius: 16px;
                border-bottom:1px solid #ccc;
                border-right:1px solid #ccc;
                border-left:1px solid #ccc;
                word-wrap: break-word;
            }
            .sf-reset .block_exception { background-color:#ddd; color: #333; padding:20px;
                -webkit-border-top-left-radius: 16px;
                -webkit-border-top-right-radius: 16px;
                -moz-border-radius-topleft: 16px;
                -moz-border-radius-topright: 16px;
                border-top-left-radius: 16px;
                border-top-right-radius: 16px;
                border-top:1px solid #ccc;
                border-right:1px solid #ccc;
                border-left:1px solid #ccc;
                overflow: hidden;
                word-wrap: break-word;
            }
            .sf-reset a { background:none; color:#868686; text-decoration:none; }
            .sf-reset a:hover { background:none; color:#313131; text-decoration:underline; }
            .sf-reset ol { padding: 10px 0; }
            .sf-reset h1 { background-color:#FFFFFF; padding: 15px 28px; margin-bottom: 20px;
                -webkit-border-radius: 10px;
                -moz-border-radius: 10px;
                border-radius: 10px;
                border: 1px solid #ccc;
            }
        </style>
    </head>
    <body>
                    <div id="sf-resetcontent" class="sf-reset">
                <h1>Whoops, looks like something went wrong.</h1>
                                        <h2 class="block_exception clear_fix">
                            <span class="exception_counter">1/1</span>
                            <span class="exception_title"><abbr title="Exception">Exception</abbr> in <a title="[snip]/src/Application.php line 243" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Application.php line 243</a>:</span>
                            <span class="exception_message">Koala</span>
                        </h2>
                        <div class="block">
                            <ol class="traces list_exception">
       <li> in <a title="[snip]/src/Application.php line 243" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Application.php line 243</a></li>
       <li>at <abbr title="Bolt\Application">Application</abbr>->initProviders() in <a title="[snip]/src/Application.php line 119" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Application.php line 119</a></li>
       <li>at <abbr title="Bolt\Application">Application</abbr>->initialize() in <a title="[snip]/src/Application.php line 64" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Application.php line 64</a></li>
       <li>at <abbr title="Bolt\Application">Application</abbr>->__construct(<em>array</em>('path_resolver_factory' => <em>object</em>(<abbr title="Closure">Closure</abbr>), 'path_resolver.root' => '[snip]', 'path_resolver.paths' => <em>array</em>('web' => '.'), 'bolt_version' => '3.3.0 alpha 8', 'bolt_name' => 'alpha', 'bolt_released' => <em>false</em>, 'bolt_long_version' => '3.3.0 alpha 8', 'deprecated.php' => <em>false</em>)) in <a title="[snip]/app/bootstrap.php line 172" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">bootstrap.php line 172</a></li>
       <li>at <abbr title=""></abbr>Bolt\{closure}()</li>
       <li>at <abbr title=""></abbr>call_user_func(<em>object</em>(<abbr title="Closure">Closure</abbr>)) in <a title="[snip]/app/bootstrap.php line 213" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">bootstrap.php line 213</a></li>
       <li>at <abbr title=""></abbr>require('[snip]/app/bootstrap.php') in <a title="[snip]/app/nut line 14" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">nut line 14</a></li>
    </ol>
</div>

            </div>
    </body>

```